challenges/1-grades/grades.js

```diff
@@ -13,6 +13,25 @@
 */
 
 function letterGrade(numberGrade) {
+    if (numberGrade < 0) {
```

Comment:
change this to if(numberGrade < 0 || numberGrade > 100) so you're not repeating the return null.